lib/puppet/parser/functions/new.rb

```diff
@@ -864,7 +866,7 @@
 | b | The data is in base64 encoding, padding as required by base64 strict is added by default
 | u | The data is in URL safe base64 encoding
 | B | The data is in base64 strict encoding
-| s | The data is a puppet string used verbatim as a binary byte sequence
+| s | The data is a puppet string seen as a binary sequence of bytes with ignored encoding
```

Comment:
It needs to be phrased so that it is understandable by a puppet user without having to understand things like forced encoding - "uses the bytes of the string, whatever they are" to use a layman's term. Will change to "seen as the verbatim byte sequence representing the string in its given encoding" in anticipation you find that more correct.